recipes/cpython/all/conanfile.py

```diff
@@ -0,0 +1,343 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, MSBuild, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import re
+
+
+class CPythonConan(ConanFile):
+    name = "cpython"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.python.org"
+    description = "GNU libtool is a generic library support script. "
```

Comment:
Looks like copy-n-pasted from another recipe :wink: (same for `topics`).

Reply:
Oops! Thanks for reviewing this!